commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/facade/PacketReader.java

```diff
@@ -53,13 +64,57 @@
     public String getField(String id, String field, String source, String process, boolean bypassCache) {
         LOGGER.info(PacketManagerLogger.SESSIONID, PacketManagerLogger.REGISTRATIONID, id,
                 "getFields for fields : " + field + " source : " + source + " process : " + process);
-        String value;
+        String value=null;;
+        if(source==null ||process==null||source.isBlank() || process.isBlank()) {

```

Comment:
if you reverse the conditon then you dont need an else. Also why cant your use StringUtils.isNoneEmpty()?